Automattic/delphin
# Contact Information: Add howdy message with gravatar

This pull request fixes #193 by displaying a hello message with possibly a Gravatar on the `Contact Information` page when it loads or as soon as the user leaves the `First Name` field:

<img width="374" alt="screenshot" src="https://cloud.githubusercontent.com/assets/594356/21591512/2e3650d2-d105-11e6-861e-41e1bbe7dfa6.png">

The Gravatar is only displayed when a valid email address has been entered.

#### Testing instructions

1. Run `git checkout add/howdy-message` and start your server, or open a [live branch](https://delphin.live/?branch=add/howdy-message)
2. Open the [`Search` page](http://delphin.localhost:1337/search) and enter search terms
3. Select a domain and log in with a test account with no contact information
4. Check that no message is displayed when the `Contact Information` page loads
5. Enter a first name and hit <kbd>tab</kbd>
6. Check that a howdy message is now displayed
7. Check that a Gravatar is displayed if the email provided has one
8. Repeat steps #2 to #7 with a test account with contact information this time

#### Reviews
